domains/_vercel.kevinmolina.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "kevinmolinadev",
+        "email": "kevinmolina.dev@gmail.com"
+    },
+    "record": {
+        "TXT": "514e23dcdfedeb5ae4c5"
```

```suggestion
        "TXT": "514e23dcdfedeb5ae4c5"
```
This is not a proper Vercel verification string. Please recheck Vercel and get the full version.